pkg/releaseutil/manifest.go

```diff
@@ -31,17 +31,19 @@ type SimpleHead struct {
 	} `json:"metadata,omitempty"`
 }
 
+var sep = regexp.MustCompile("(?:^|\\s*\n)---\\s*")
```

Comment:
In the interest of full disclosure, I'll note that it's _possible_—though I don't think Kubernetes does so—that the YAML stream could also include a positive number of [_ending_ document boundary markers](http://yaml.org/spec/current.html#c-document-end) ("...") before the starting one ("---"). Accommodating those—including any intervening comments that would be valid—with a regular expression is impractical.